instances/experimentalApproach/biochemistry.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/experimentalApproach/biochemistry",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/ExperimentalApproach",
+  "definition": "'Biochemistry' covers any experimental approach that studies chemical processes within and relating to living organisms. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Biochemistry)]",
```

Comment:
For consistency I would call it `biochemical approach` with `biochemistry` as a synonym to avoid conflation of the field of study with the experimental approach.

Reply:
@tgbugs this would be in conflict with how we named all experimental approaches in openMINDS (e.g., histology, electrophysiology etc). In principle we use a mixed interpretation in openMINDS (something between field of study and exp. approach).

But I definitely should add biochemical approach to the synonyms (same for culturing)